cpu/atmega_common/include/arduino_arch.hpp

```diff
@@ -0,0 +1,281 @@
+/*
+  arduino_arch.hpp - Pin definition functions for Arduino
+  Part of Arduino - http://www.arduino.cc/
+
+  Copyright (c) 2007 David A. Mellis
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General
+  Public License along with this library; if not, write to the
+  Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+  Boston, MA  02111-1307  USA
+
+  NOTE: The code in this file comes from https://github.com/arduino/ArduinoCore-avr
+  (see ArduinoCore-avr/variants/.../pins_arduino.h)
+  But has been modified to follow RIOT's coding style.
+*/
+
+/**
+ * @ingroup    cpu_atmega_common
+ * @{
+ *
+ * @file
+ * @brief      Compatibility with (AVR specific) Arduino libraries
+ */
+
+#ifndef ARDUINO_ARCH_H
+#define ARDUINO_ARCH_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if defined(BOARD_ARDUINO_LEONARDO)
+
+/**
+ * @brief   Return the PCICR register for the given digital pin
+ */
+static inline volatile uint8_t *digitalPinToPCICR(int pin)
+{
+    if (((pin >= 8) && (pin <= 11))
+        || ((pin >= 14) && (pin <= 17))
+        || ((pin >= ARDUINO_PIN_A8) && (pin <= ARDUINO_PIN_A10)))
+        {
+        return &PCICR;
+    }
+    return NULL;
+}
+
+/**
+ * @brief   Return the bit in the PCICR register for the given digital pin
+ */
+static inline uint8_t digitalPinToPCICRbit(int pin)
+{
+    (void)pin;
+    return 0;
+}
+
+/**
+ * @brief   Return the PCMSK register for the given digital pin
+ */
+static inline volatile uint8_t *digitalPinToPCMSK(int pin)
+{
+    if (((pin >= 8) && (pin <= 11))
+        || ((pin >= 14) && (pin <= 17))
+        || ((pin >= ARDUINO_PIN_A8) && (pin <= ARDUINO_PIN_A10)))
+        {
+        return &PCMSK0;
+    }
+    return NULL;
+}
+
+/**
+ * @brief   Return the bit in the PCMSK register for the given digital pin
+ */
+static inline uint8_t digitalPinToPCMSKbit(int pin)
+{
+    if ((pin >= 8) && (pin <= 11)) {
+        return pin - 4;
+    } else if (pin == 14) {
+        return 3;
```

Comment:
The [coding conventions](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#indentation-and-braces) states that `if/else/else if` should look like this:
```suggestion
    if ((pin >= 8) && (pin <= 11)) {
        return pin - 4;
    }
    else if (pin == 14) {
        return 3;
```

Maybe run uncrustify on this file ?